### ISSUE TITLE
Card content overflows card container

### ISSUE THREAD

**Issue by seanforyou23** [patternfly/patternfly-3#836]
While working with the info status card, it was realized that when the card container becomes too small, the textual contents will overflow the parent element.

Below image illustrates the issue;

<img width="971" alt="screen shot 2017-11-09 at 9 16 04 am" src="https://user-images.githubusercontent.com/5942899/32630778-e733d57c-c56b-11e7-8957-d37ec146bf62.png">

You can see it in action from the following link, just resize the browser window down to a small width;

https://rawgit.com/seanforyou23/patternfly-ng/info-card-example-dist/dist-demo/#/infocard

I began an attempt to [fix this issue](https://github.com/seanforyou23/patternfly/commit/0c1593705ee069bce5ea0abf4535a2edaf16a9fb), but then realized the word-break: break-word; and overflow-wrap: break-word; style definitions do not fix this issue for the card component in firefox.

**Comment by michpetrov** [comment]
@seanforyou23 I think this can be fixed by setting proper classes for the column grid (and setting a `min-width` for the container); that way the cards will never get too narrow and overflowing text won't be an issue.

**Comment by jeff-phillips-18** [comment]
@seanforyou23 where are you seeing this?

**Comment by jeff-phillips-18** [comment]
@seanforyou23 Sorry, just saw the link. This looks like a patternfly-ng issue and should be opened against patternfly-ng.

**Comment by seanforyou23** [comment]
@michpetrov thank you for the suggestion - I did try this but found that it didn't yield the best experience in many cases. I ended up achieving the desired result by using the word-break property. By doing it this way, I was able to go from this...
<img width="246" alt="screen shot 2017-11-15 at 1 27 53 pm" src="https://user-images.githubusercontent.com/5942899/32859489-a8775920-ca1c-11e7-9577-16a6c55e3b8e.png">
To this...
<img width="201" alt="screen shot 2017-11-15 at 1 16 34 pm" src="https://user-images.githubusercontent.com/5942899/32859510-b63a1160-ca1c-11e7-8474-15d06b931add.png">
_At least as a default behavior_ I think it may be more flexible, then of course an individual application could (and probably would) apply a min-width to tailor the behavior to their liking.

@jeff-phillips-18 sorry I was initially thinking it was an issue with the core styles because I didn't realize yet that it was acceptable to apply custom styling to the examples in patternfly-ng. With this in mind I was able to easily apply the styling I needed to make the examples look good.
